src/presign/participant.rs

```diff
@@ -1340,12 +1340,12 @@ mod test {
         errors::{CallerError, InternalError, Result},
         paillier::DecryptionKey,
         ring_pedersen::VerifiedRingPedersen,
-        utils::{k256_order, testing::init_testing},
+        utils::testing::init_testing,
         AuxInfoPrivate, AuxInfoPublic, Identifier, KeySharePrivate, KeySharePublic,
         ParticipantConfig, ParticipantIdentifier, PresignParticipant, ProtocolParticipant,
     };
-    use libpaillier::unknown_order::BigNumber;
-    use rand::rngs::StdRng;
+    use ::rand::rngs::StdRng;
+    //use rug::rand;
```

Comment:
nit: in L1347, you shouldn't need the preceding `::`. Remove commented-out L1348.